### PR TITLE
Correct README based on feedback of issue #1374

### PR DESCRIPTION
**Pull request by shivathapaa**
# Overview
This pull request addresses feedback (PR #1376) and makes corrections to the README file based on the discussions in issue #1374.

# Related Issue / Discussion
As per feedback based on issue: #1374
- Made corrections to the README file to address the feedback and suggestions provided.
- Improved the introduction to accurately describe the role of `pygeoapi`.
- Included a link to the SensorThings standard for additional reference.
- Corrected the number of available STA examples, and moved the Docker Compose description to _Additional Information_.

These changes resolve issue #1374 and ensure that the README accurately reflects the project's functionality.

# Additional Information
- I changed the title for more clarity, "Setting up a Pygeoapi Server with SensorThings (STA) Using Docker Compose"
- Feel free to provide feedback or ask for any clarifications if needed.

# Contributions and Licensing

(as per https://github.com/geopython/pygeoapi/blob/master/CONTRIBUTING.md#contributions-and-licensing)

- [x] I'd like to contribute [feature X|bugfix Y|docs|something else] to pygeoapi. I confirm that my contributions to pygeoapi will be compatible with the pygeoapi license guidelines at the time of contribution.
- [x] I have already previously agreed to the pygeoapi Contributions and Licensing Guidelines

## Acknowledgments:
Thanks, @doublebyte1, for your continued support. I've resolved the issue by closing and reopening the pull request. Apologies for any inconvenience, but this has been a valuable learning for me. :)

### CODE REVIEW

File: docker/examples/sensorthings/README.md [hunk 1]
```diff
@@ -1,31 +1,51 @@
-# Pygeoapi with SensorThings (STA)
+# Setting up a Pygeoapi Server with SensorThings (STA) Using Docker Compose
```
Comment (tomkralidis) [inline docker/examples/sensorthings/README.md:1]: Good opportunity to replace `Pygeoapi` with `pygeoapi`! Can we adjust to:

Setting up a pygeoapi server with SensorThings (STA) using Docker Compose

**Review by tomkralidis — CHANGES_REQUESTED** [review]

File: docker/examples/sensorthings/README.md (hunk 1 above)
Comment (webb-ben) [reply docker/examples/sensorthings/README.md:1]: +1

Comment (doublebyte1) [reply docker/examples/sensorthings/README.md:1]: @shivathapaa do you want to make the proposed change?

Comment (shivathapaa) [reply docker/examples/sensorthings/README.md]: Thank you for the feedback! I have made changes accordingly. :)

**Pull request merged**